pkg/config/application_kubernetes_test.go

```diff
@@ -97,7 +97,8 @@ func TestKubernetesApplicationConfig(t *testing.T) {
 					},
 				},
 				Input: KubernetesDeploymentInput{
-					AutoRollback: newBoolPointer(true),
+					AutoRollback:        newBoolPointer(true),
+					AutoCreateNamespace: newBoolPointer(false),
```

Comment:
```suggestion
					AutoCreateNamespace: false,
```
or just remove it, due to the default value is set to false for the bool type in Golang.

Reply:
yeah, got it as mentioned above! thanks